index.js

```diff
@@ -1,13 +1,15 @@
 const express = require('express')
-const bodyParser = require('body-parser')
 const movies = require('./movies')
 const { response } = require('express')
+const { getAllMovies, getMovieByTitle, getMovieByDirectors } = require('./controller/movies')
 
 const app = express()
 
-app.get('/', (request, response) => {
-  return response.send(movies)
-})
+app.get('/', getAllMovies)
```

Comment:
In the README it asks you to deliver the movies to a specific route so make sure you are looking for that.